tables/chromeuserprofiles/chrome_user_profiles.go

```diff
@@ -77,18 +78,31 @@ func generateForPath(ctx context.Context, fileInfo userFileInfo) ([]map[string]s
 		return nil, errors.Wrap(err, "unmarshalling chome local state")
 	}
 
-	for _, profileInfo := range localState.Profile.InfoCache {
+	for profileDir, profileInfo := range localState.Profile.InfoCache {
 		results = append(results, map[string]string{
 			"username":  fileInfo.user,
 			"email":     profileInfo.Email,
 			"name":      profileInfo.Name,
 			"ephemeral": strconv.Itoa(btoi(profileInfo.Ephemeral)),
+			"path":      profilePathIfExists(fileInfo.path, profileDir),
 		})
 	}
 
 	return results, nil
 }
 
+func profilePathIfExists(localStatePath, profileDir string) string {
+	localStateDir := filepath.Dir(localStatePath)
+	profilePath := filepath.Join(localStateDir, profileDir)
+	if _, err := os.Stat(profilePath); err != nil {
+		// If there's an error of any kind, assume the profile path doesn't
```

Comment:
Perhaps it would be better to handle the specific not found error and surface other errors?

Reply:
I've propagated the error up to the `generateForPath` function, but I've noticed that `GoogleChromeProfilesGenerate` currently ignores all errors returned from all of the functions it calls.

If we surface the errors returned from my changes, I think we should also surface the other possible errors from the functions called. But this would be a change in how this table works, from never returning errors in any scenario, to returning errors in some scenarios.

Thoughts?